### PR TITLE
Added enumerations for BrondatumArchiefprocedure_Afleidingswijze,

### PR DESCRIPTION
**Pull request by joeribekker**
Archiefnominatie and Archiefstatus

### CODE REVIEW

File: zds_schema/constants.py [hunk 1]
```diff
@@ -67,3 +67,93 @@ class RolTypes(DjangoChoices):
...
+        _("Het zaakdossier moet bewaard blijven en op de Archiefactiedatum "
+          "overgedragen worden naar een archiefbewaarplaats.")
+    )
+    vernietigen = ChoiceItem('vernietigen',
+        _("Het zaakdossier moet op of na de Archiefactiedatum vernietigd "
+          "worden.")
+    )
+
+class Archiefstatus(DjangoChoices):
+    nog_te_archiveren = ChoiceItem('nog_te_archiveren',
```
Comment (sergei-maertens) [inline zds_schema/constants.py]:
```suggestion
    nog_te_archiveren = ChoiceItem(
        'nog_te_archiveren',
```

File: zds_schema/constants.py [hunk 2]
```diff
@@ -67,3 +67,93 @@ class RolTypes(DjangoChoices):
...
+        _("De zaak cq. het zaakdossier is nog niet als geheel gearchiveerd.")
+    )
+    gearchiveerd = ChoiceItem('gearchiveerd',
```
Comment (sergei-maertens) [inline zds_schema/constants.py]:
```suggestion
    gearchiveerd = ChoiceItem(
        'gearchiveerd',
```

File: zds_schema/constants.py [hunk 3]
```diff
@@ -67,3 +67,93 @@ class RolTypes(DjangoChoices):
...
+          "bewaarbaar gemaakt maar de vernietigingsdatum kan nog niet bepaald "
+          "worden.")
+    )
+    # After deliberation this element was removed because "vernietigd" means
+    # it's really gone and the status wouldn't make sense:
+    #
+    # vernietigd = ChoiceItem('vernietigd',
+    #     _("De zaak cq. het zaakdossier is vernietigd.")
+    # )
+    overgedragen = ChoiceItem('overgedragen',
```
Comment (sergei-maertens) [inline zds_schema/constants.py]:
```suggestion
    overgedragen = ChoiceItem(
        'overgedragen',
```

File: zds_schema/constants.py [hunk 4]
```diff
@@ -67,3 +67,93 @@ class RolTypes(DjangoChoices):
...
+        _("De zaak cq. het zaakdossier is overgebracht naar een "
```
Comment (sergei-maertens) [inline zds_schema/constants.py]: PEP8 is setup to allow for 120 char line lengths, so I think this fits a single line

File: zds_schema/constants.py [hunk 5]
```diff
@@ -67,3 +67,93 @@ class RolTypes(DjangoChoices):
...
+          "archiefbewaarplaats.")
+    )
+
+
+class BrondatumArchiefprocedure_Afleidingswijze(DjangoChoices):
```
Comment (sergei-maertens) [inline zds_schema/constants.py]: that underscore is a bit ugly, no?

File: zds_schema/constants.py [hunk 6]
```diff
@@ -67,3 +67,93 @@ class RolTypes(DjangoChoices):
...
+    afgehandeld = ChoiceItem('afgehandeld',
```
Comment (sergei-maertens) [inline zds_schema/constants.py]:
```suggestion
    afgehandeld = ChoiceItem(
        'afgehandeld',
```

File: zds_schema/constants.py [hunk 7]
```diff
@@ -67,3 +67,93 @@ class RolTypes(DjangoChoices):
...
+        _("De termijn start op de datum waarop de zaak is afgehandeld "
+          "(ZAAK.Einddatum in het RGBZ).")
+    )
+    ander_datumkenmerk = ChoiceItem('ander_datumkenmerk',
```
Comment (sergei-maertens) [inline zds_schema/constants.py]:
```suggestion
    ander_datumkenmerk = ChoiceItem(
        'ander_datumkenmerk',
```

File: zds_schema/constants.py [hunk 8]
```diff
@@ -67,3 +67,93 @@ class RolTypes(DjangoChoices):
...
+        _("De termijn start op de datum die is vastgelegd in een ander "
+          "datumveld dan de datumvelden waarop de overige waarden (van deze "
+          "attribuutsoort) betrekking hebben. Objecttype, Registratie en "
+          "Datumkenmerk zijn niet leeg.")
+    )
+    eigenschap = ChoiceItem('eigenschap',
```
Comment (sergei-maertens) [inline zds_schema/constants.py]:
```suggestion
    eigenschap = ChoiceItem(
        'eigenschap',
```

File: zds_schema/constants.py [hunk 9]
```diff
@@ -67,3 +67,93 @@ class RolTypes(DjangoChoices):
...
+        _("De termijn start op de datum die vermeld is in een "
+          "zaaktype-specifieke eigenschap (zijnde een `datumveld`).	"
+          "`ResultaatType.ZaakType` heeft een `Eigenschap`; `Objecttype`, en "
+          "`Datumkenmerk` zijn niet leeg.")
+    )
+    gerelateerde_zaak = ChoiceItem('gerelateerde_zaak',
```
Comment (sergei-maertens) [inline zds_schema/constants.py]:
```suggestion
    gerelateerde_zaak = ChoiceItem(
        'gerelateerde_zaak',
```

File: zds_schema/constants.py [hunk 10]
```diff
@@ -67,3 +67,93 @@ class RolTypes(DjangoChoices):
...
+        _("De termijn start op de datum waarop de gerelateerde zaak is "
+          "afgehandeld (ZAAK.Einddatum of ZAAK.Gerelateerde_zaak.Einddatum in "
+          "het RGBZ). `ResultaatType.ZaakType` heeft gerelateerd `ZaakType`")
+    )
+    hoofdzaak = ChoiceItem('hoofdzaak',
```
Comment (sergei-maertens) [inline zds_schema/constants.py]:
```suggestion
    hoofdzaak = ChoiceItem(
        'hoofdzaak',
```

File: zds_schema/constants.py [hunk 11]
```diff
@@ -67,3 +67,93 @@ class RolTypes(DjangoChoices):
...
+        _("De termijn start op de datum waarop de gerelateerde zaak is "
+          "afgehandeld, waarvan de zaak een deelzaak is (ZAAK.Einddatum van de "
+          "hoofdzaak in het RGBZ).ResultaatType.ZaakType is deelzaaktype van "
+          "ZaakType")
+    )
+    ingangsdatum_besluit = ChoiceItem('ingangsdatum_besluit',
```
Comment (sergei-maertens) [inline zds_schema/constants.py]:
```suggestion
    ingangsdatum_besluit = ChoiceItem(
        'ingangsdatum_besluit',
```

File: zds_schema/constants.py [hunk 12]
```diff
@@ -67,3 +67,93 @@ class RolTypes(DjangoChoices):
...
+        _("De termijn start op de datum waarop het besluit van kracht wordt "
+          "(BESLUIT.Ingangsdatum in het RGBZ).	ResultaatType.ZaakType heeft "
+          "relevant BesluitType")
+    )
+    termijn = ChoiceItem('termijn',
```
Comment (sergei-maertens) [inline zds_schema/constants.py]:
```suggestion
    termijn = ChoiceItem(
        'termijn',
```

File: zds_schema/constants.py [hunk 13]
```diff
@@ -67,3 +67,93 @@ class RolTypes(DjangoChoices):
...
+        _("De termijn start een vast aantal jaren na de datum waarop de zaak "
+          "is afgehandeld (ZAAK.Einddatum in het RGBZ).")
+    )
+    vervaldatum_besluit = ChoiceItem('vervaldatum_besluit',
```
Comment (sergei-maertens) [inline zds_schema/constants.py]:
```suggestion
    vervaldatum_besluit = ChoiceItem(
        'vervaldatum_besluit',
```

File: zds_schema/constants.py [hunk 14]
```diff
@@ -67,3 +67,93 @@ class RolTypes(DjangoChoices):
...
+        _("De termijn start op de dag na de datum waarop het besluit vervalt "
+          "(BESLUIT.Vervaldatum in het RGBZ). ResultaatType.ZaakType heeft "
+          "relevant BesluitType")
+    )
+    zaakobject = ChoiceItem('zaakobject',
```
Comment (sergei-maertens) [inline zds_schema/constants.py]:
```suggestion
    zaakobject = ChoiceItem(
        'zaakobject',
```

File: zds_schema/constants.py (hunk 5 above)
Comment (joeribekker) [reply zds_schema/constants.py]: Yes :( suggestions?

File: zds_schema/constants.py [hunk 15]
```diff
@@ -67,3 +67,93 @@ class RolTypes(DjangoChoices):
 class ObjectTypes(DjangoChoices):
     besluit = ChoiceItem('besluit', _("Besluit"))
     zaak = ChoiceItem('zaak', _("Zaak"))
+
+
+
+class Archiefnominatie(DjangoChoices):
```
[the remark this answers is not in the document]
Comment (joeribekker) [reply zds_schema/constants.py]: See #1

**Pull request merged**